game/client.html

```diff
@@ -212,16 +214,15 @@ <h5 id="uiconsole">Console Text</h5>
             <div class="form-group">
                 <label class="col-md-4 control-label" for="Firstonewon"></label>
                 <div class="col-md-4">
-                    <button onclick="winner(0)" id="Firstonewon"
-                        class="btn btn-primary">Player 0 Wins!</button>
+                    <button onclick="winner(0)" id="Firstonewon" class="btn btn-primary">Player 0 Wins!</button>
```

Comment:
This goes for most of the code, but we need to ditch these static onclick functions soon